core-library/src/main/java/org/silverpeas/core/socialnetwork/relationship/RelationShipDao.java

```diff
@@ -272,11 +261,6 @@ private RelationShip resultSet2RelationShip(ResultSet rs) throws SQLException {
     return relationShip;
   }
 
-  /**
-   * convert list of contact ids to string for using in the query SQL
-   * @param list
-   * @return String
-   */
   private static String toSqlString(List<String> list) {
     StringBuilder result = new StringBuilder(100);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Assign this magic number 100 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)